manifests/init.pp

```diff
@@ -187,7 +191,8 @@
 
   if $install_java {
     class {'java':
-      distribution => 'jdk'
+      distribution          => 'jdk',
```

Comment:
Yeah, I'm not sure why I went so overboard here, should've just made the arrows match for linting. Let me fix.